### DISCUSSION TITLE
how to disable automatically opening browser during dev cmd?

### DISCUSSION THREAD

**Discussion by ArnavK-09** [wxt-dev/wxt#1004] [Q&A] [answered]
on 'npm run dev' wxt

**Comment by 1natsu172** [comment]
@ArnavK-09 https://github.com/wxt-dev/wxt/blob/efcb593718edf9c648c6dd4876f23a180097550e/packages/wxt/src/types.ts#L920

setup to `wxt.config.ts` manifest,

```ts
  manifest: {
    runner: {
      disabled: true
    },
}
```

**Reply by ArnavK-09** [reply]
altho that's the solution ig
but didn't worked with mv3
nvm thanks! ill use nodemon

**Reply by ArnavK-09** [reply]
![image](https://github.com/user-attachments/assets/234d401a-38f6-42df-a1b2-05efef73b59f)

**Comment by Nithur-M** [comment, marked as answer]
Hi @ArnavK-09,

If you are still looking for the solution, add this to to the `defineConfig` function in `wxt.config.ts` file:
```
webExt: {
  disabled: true,
},
```

**Comment by PatrykKuniczak** [comment]
@ArnavK-09 Let's mark this as "answered", if you fixed it :)
